scripts/prepare_automatic_backups.py

```diff
@@ -80,21 +94,36 @@ def update_cron_dict(cron_dict):
 
 
 def get_cron_dict():
+    """Converting  yaml file content that is yaml type into dictionary type.
+    
+    Returns:
+      type=dictionary. returning yaml file content in dictionary content.
+    """
     return utils.dict_from_yaml(utils.get_file_contents(_CRON_YAML_FILE_NAME))
 
 
 def save_cron_dict(cron_dict):
+    """Converting dictionary content into yaml and saving into cron_yaml_file(filename) 
+    file.
+    
+    Args:
+      cron_dict type=dictionary. it is the content of  
+    """
     with open(_CRON_YAML_FILE_NAME, 'wt') as cron_yaml_file:
         cron_yaml_file.write(utils.yaml_from_dict(cron_dict))
 
 
 def update_yaml_files():
+    """Updating yaml fies
+    """
```

Comment:
Please add period (.) at the end and close with """ in the same line as it is just one line docstring.

Reply:
resolved it.